plugins/search-backend-module-explore/README.md

```diff
@@ -39,3 +39,18 @@ backend.add(searchPlugin());
 backend.add(searchModuleExploreCollator({ schedule }));
 backend.start();
 ```
+
+### Using Auth Middleware
+
+If your Backstage instance uses service-to-service authentication you can pass an optional tokenManager to the collator factory. This will ensure that the collator makes authenticated requests to the explore backend.
```

Comment:
If you put backticks around `tokenManager` then Vale will be happy ;)

Reply:
I'm just curious what/who is Vale? 👀